shuup/campaigns/admin_module/forms/_basket_conditions.py

```diff
@@ -65,6 +65,11 @@ class Meta(BaseRuleModelForm.Meta):
         model = BasketMaxTotalAmountCondition
 
 
+class ChildrenProductConditionForm(BaseRuleModelForm):
+    class Meta(BaseRuleModelForm.Meta):
+        model = ChildrenProductCondition
```

Comment:
I guess here you need to filter product options to be from the current shop and also from the current supplier, if that exists